bower.json

```diff
@@ -19,17 +19,35 @@
   },
   "main": "paper-toast.html",
   "dependencies": {
-    "iron-a11y-announcer": "PolymerElements/iron-a11y-announcer#^1.0.0",
-    "iron-overlay-behavior": "PolymerElements/iron-overlay-behavior#^1.0.9",
-    "iron-fit-behavior": "PolymerElements/iron-fit-behavior#^1.1.0",
-    "polymer": "Polymer/polymer#^1.5.0"
+    "iron-a11y-announcer": "PolymerElements/iron-a11y-announcer#2.0-preview",
+    "iron-overlay-behavior": "PolymerElements/iron-overlay-behavior#2.0-preview",
+    "polymer": "Polymer/polymer#^2.0.0-rc.1"
```

Comment:
`iron-overlay-behavior 1.x` depends on `iron-fit-behavior ^1.0.0`, while `paper-toast` requires `iron-fit-behavior 1.1.x` or superior ([this PR](https://github.com/PolymerElements/paper-toast/pull/74) introduced the dependency on this specific version).
In 2.x, `iron-overlay-behavior` depends on `iron-fit-behavior 2.x` so `paper-toast` doesn't need to specify that anymore.